modules/src/ics04_channel/msgs.rs

```diff
@@ -1,7 +1,6 @@
 #![allow(clippy::too_many_arguments)]
 use super::channel::{ChannelEnd, Counterparty, Order};
 
-use crate::ics03_connection::connection::validate_version;
```

Comment:
Following the [recent PR ](https://github.com/informalsystems/ibc-rs/commit/49129e591e5555837de69d7959d3b12a39e2e37a), there may be major conflicts in file `modules/src/ics04_channel/msgs.rs`, preventing the merge. From what I can tell, the modifications here are no longer relevant, so it's safe to choose the quick fix of leaving this file as it is on the master currently: https://github.com/informalsystems/ibc-rs/blob/master/modules/src/ics04_channel/msgs.rs (choose merging strategy `theirs` I think).

PS: The changes here are small: basically you're removing any use of the `ICS03::validate_version` method from what I can tell, and I'm doing the same changes in #391, so we will not lose these fixes.